### DISCUSSION TITLE
use wrap balancer caused hydration error on nextJS

### DISCUSSION THREAD

**Discussion by fiqryq** [shuding/react-wrap-balancer#17] [Q&A] [answered]
Hello, guys, can you help me I always got a hydration error when I use wrap balancer.

![image](https://user-images.githubusercontent.com/25787603/210662324-3dbd147e-c033-4435-ae68-dcda03c9599b.png)

**Comment by shuding** [comment, marked as answer]
This should have been fixed in latest version.

**Reply by fiqryq** [reply]
cool thanks!!
